runtime/primitives/src/currency.rs

```diff
@@ -9,12 +70,13 @@ use sp_runtime::{ArithmeticError, DispatchError, RuntimeDebug};
 use serde::{Deserialize, Serialize};
 use sp_runtime::sp_std::ops::Deref;
 
+/// `MayBe`CurrencyId as not each `u128` is valid id.
```

Comment:
should we rename struct from `CurrencyId` to `Maybe...` ?

Reply:
Not in this PR I think. Too many changes will appear.